tests/test_corpora_reddit_reader.py

```diff
@@ -0,0 +1,63 @@
+# from __future__ import absolute_import, unicode_literals
```

Comment:
I would just completely drop this file for now instead of leaving it in commented out.